docker/build_testing_sheet.dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:bionic
+
+RUN apt-get update -y
+
+COPY . /kolibri
```

Comment:
Is the entire Kolibri repo necessary?  It's likely I overlooked something, but the script seems to pull a lot of its data from the github directly, anyway. Copying in the entire repo this early in the stage makes for a lot of duplicated layers. They end up getting cleaned later in the pipeline, but reducing the amount of copies could have significant speed implications.

Reply:
Thanks for that information @DXCanas I'll copy on what the script needed in the `./kolibri ` directory.